regression/CMakeLists.txt

```diff
@@ -124,6 +124,7 @@ set(REGRESSIONS_CPP11 esbmc-cpp11/cpp
                       esbmc-cpp11/template
                       esbmc-cpp11/tuple
                       esbmc-cpp11/array
+                      esbmc-cpp11/lambda
```

Comment:
> Nice, what happened with the new failing cases?

They failed in the first place.CI didn't check them because it didn't put CMakeLists in.